docs/libraries/common-dependency-injection-autofac.md

```
@@ -0,0 +1,62 @@
+# Ploch.Common.DependencyInjection.Autofac
+
+> Autofac container integration for the ServicesBundle pattern.
+
```

markdownlint-cli2 (0.21.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)